docs/development.md

```diff
@@ -128,6 +128,13 @@ To conveniently run all the tests you can run the lein alias
 ```
 lein alltests
 ```
+### Automated accessibility test report
```

Comment:
did you mean to put this here? now "Automated accessibility test report" is under "Development database" which sounds a bit weird

Reply:
Isnt is under Running tests now?

Reply:
you're right! I'm blind.